GiniVision/Classes/Core/Extensions/Collection.swift

```diff
@@ -8,10 +8,34 @@
 
 import Foundation
 
+extension Collection {
+    var isNotEmpty: Bool {
```

Comment:
What's the point? You don't want to have double negatives?

Reply:
To make it more clear in the code, instead of having the likely missable `!` before of `isEmpty`

Reply:
I personally don't see a big benefit, but if you like it, ok.